package.json

```diff
@@ -33,12 +34,14 @@
     "warning": "^4.0.3"
   },
   "devDependencies": {
-    "chai": "^4.3.4",
+    "aud": "^2.0.0",
+    "chai": "^3.5.0",
```

Comment:
Why go from chai v4 to v3?

Reply:
because chai v4 doesn't support all the node versions that this package currently does; it shouldn't have been upgraded in the first place (i made the mistake in baf08111)